scripts/setup.py

```diff
@@ -231,22 +232,31 @@ def __init__(self, p):
         self.includes = ['lib', 'test', 'test_driver',
                          'integration_test', 'pubspec.yaml', 'README.md']
 
-    def get_value_in_pubspec_file(self, key):
-        pubspec_file = self.project.project_path + os.sep + "pubspec.yaml"
-        f = open(pubspec_file, "r")
+    def get_value_in_yaml_file(self, filename, key):
+        yaml_file = self.project.project_path + os.sep + filename
+        f = open(yaml_file, "r")
         file_text = f.read()
         f.close()
         for line in file_text.split("\n"):
             if key in line.strip():
                 return line.strip().replace(key, "").strip()
         return None
 
+    def get_value_in_pubspec_file(self, key):
+        return self.get_value_in_yaml_file("pubspec.yaml", key)
+
+    def get_value_in_build_file(self, key):
+        return self.get_value_in_yaml_file("build.yaml", key)
+
     def get_old_project_name(self):
         return self.get_value_in_pubspec_file("name:")
 
     def get_current_project_version(self):
         return self.get_value_in_pubspec_file("version:")
 
+    def get_json_serializable_field_rename(self):
+        return self.get_value_in_build_file("field_rename:")
```

Comment:
please add `TODO` comment for writing unit test for this function later, maybe after we fixed the failed unit tests 😄